include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
You have suggested one fix, I'd hope for a different one :-) Can you tell me what the error looks like? I can try to provide a separate fix for that.